### ISSUE TITLE
Issues about related work and dataset

### ISSUE THREAD

**Issue by Ethan00Si** [17231087/UDITSR#1]
Hi, @17231087 

  Glad to see your papaer being accepted. 
  However, I noticed that you mentioned that `We collected this dataset because there was no public dataset that includes both search and recommendation data'. Actually, there is a dataset, namely [KuaiSAR](https://kuaisar.github.io/), satisfying the condition. Additionally, this paper seems being lack of citations of related works on joint modeling of search and recommendation. Hope you can consider citing following highly related papers: When Search Meets Recommendation: Learning Disentangled Search Representation for Recommendation SIGIR 22, UniSAR: Modeling User Transition Behaviors between Search and Recommendation SIGIR 23, A Model-Agnostic Causal Learning Framework for Recommendation using Search Data WWW 2022, USER: A Unified Information Search and Recommendation Model based on Integrated Behavior Sequence CIKM 21.

  In summary, hope you can consider adding the mentioned works.

**Comment by Ethan00Si** [comment]
Unfortunately, three weeks have passed and I haven't received any reply from the authors.
